src/app/containers/ConsentBanner/Canonical/logic/index.js

```diff
@@ -0,0 +1,91 @@
+import 'isomorphic-fetch';
+import Cookie from 'js-cookie';
+import getCookieOvenBaseUrl from './cookieOvenBaseUrl';
+
+const PRIVACY_COOKIE = 'ckns_privacy';
+const EXPLICIT_COOKIE = 'ckns_explicit';
+const POLICY_COOKIE = 'ckns_policy';
+const COOKIE_EXPIRY = 365;
+const BANNER_APPROVED = '1';
+const POLICY_APPROVED = '111';
+const POLICY_DENIED = '000';
+
+const onClient = typeof window !== 'undefined';
+
+const setCookie = (name, value) =>
+  Cookie.set(name, value, { expires: COOKIE_EXPIRY });
+
+const setCookieOven = value => {
+  if (window.location && window.location.origin) {
+    try {
+      fetch(
+        `${getCookieOvenBaseUrl(
+          window.location.origin,
+        )}/${POLICY_COOKIE}/${value}`,
+      );
+    } catch (e) {} // eslint-disable-line no-empty
```

Comment:
If we error on the request to set the cookie value, we should definitely log it. One of the inputs to this file should be a logger, and if not passed in, it defaults to console.log.

Reply:
Done (but tests are WIP)